cdap-docs/admin-manual/source/security.rst

```diff
@@ -450,16 +449,17 @@ will use ``https`` instead of ``http``.
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``::
+- Submit a username and password to one of the authentication URIs (``<auth-uri>``) to
+  obtain an ``AccessToken``::
 
-    GET -u <username>:<password> <auth-uri>
+    GET <auth-uri> <username>:<password>
```

Comment:
You do for the HTTP curl command, but this (intended) to be the API, and not the command...there doesn't seem to be a standard way of describing this except as:
```
<http-method> <uri> <options, headers, body>
```